flytekit/types/structured/basic_dfs.py

```diff
@@ -103,3 +105,6 @@ def decode(
 StructuredDatasetTransformerEngine.register(ParquetToPandasDecodingHandler())
 StructuredDatasetTransformerEngine.register(ArrowToParquetEncodingHandler())
 StructuredDatasetTransformerEngine.register(ParquetToArrowDecodingHandler())
+
+StructuredDatasetTransformerEngine.register_renderer(pd.DataFrame, TopFrameRenderer())
+StructuredDatasetTransformerEngine.register_renderer(pa.Table, ArrowRenderer())
```

Comment:
could you add a quick unit test to test that these are loaded into the Renderers dict by default?

Reply:
I did. [here](https://github.com/flyteorg/flytekit/pull/1140/files#diff-7a18af2403e31dfcd1cf7250072b3f605cc31fc8c7e805ffdcf5bd88bc5392b5R427-R428)